docs/API/buttons.md

```diff
@@ -1,68 +1,68 @@
-Buttons take a title and an optional [material icon name](https://design.google.com/icons/), as well as the props below.
-
-> You can override Material icons with one of the following: material-community, simple-line-icon, zocial, font-awesome, octicon, ionicon, foundation, evilicon, or entypo by providing an icon.type as a prop.
+Buttons can be used to interact with the screen. It takes the following props.
 
 ![Buttons](http://i.imgur.com/CVf1xbr.png)
```

Comment:
we will need to update this screenshot with the new buttons layout. You could just take a screenshot from the example expo app.

Reply:
which buttons layout ?

Reply:
The one on the React Native Elements App. Have seen that yet? Here is the repo: https://github.com/react-native-training/react-native-elements-app and here is the expo app you can run on your phone: https://expo.io/@monte9/react-native-elements-app

if you run that locally, you will see that it has a buttons tab with all the types of buttons displayed on it. I suggest you screenshot that put together something presentable.

Reply:
If this sounds like too much work, don't worry about it. I'll follow up on it.

Reply:
Added this